telegram/ext/conversationhandler.py

```diff
@@ -136,6 +141,9 @@ class ConversationHandler(Handler[Update]):
         map_to_parent (Dict[:obj:`object`, :obj:`object`], optional): A :obj:`dict` that can be
             used to instruct a nested conversationhandler to transition into a mapped state on
             its parent conversationhandler in place of a specified nested state.
+        run_async (:obj:`bool`, optional): If set to :obj:`True`, all the handlers in this
+            :class:`ConversationHandler` will be set with :attr:`run_async` equals to :obj:`True`.
```

Comment:
The wording is somewhat hard to understand (same below). suggestion:

```suggestion
        run_async (:obj:`bool`, optional): Pass :obj:`True` to override the :attr:`Handler.run_async` setting of all handlers (in :attr:`entry_points`, :attr:`states` and :attr:`fallbacks:`.
```

I guess this formulation would also make above note unnecessary …

Reply:
Updated with new commits